rest/main.go

```diff
@@ -49,8 +49,8 @@ func serverMain(ctx context.Context, osArgs []string) error {
 	adminInterfaceAuthFlag := fs.Bool("api.admin_interface_authentication", false, "")
 	metricsInterfaceAuthFlag := fs.Bool("api.metrics_interface_authentication", false, "")
 
-	allowInsecureServerConnections := fs.Bool("bootstrap.allow_insecure_server_connections", false, "")
-	allowInsecureTLSConnections := fs.Bool("api.https.allow_insecure_tls_connections", false, "")
+	useTLSServer := fs.Bool("bootstrap.use_tls_server", true, "")
```

Comment:
Wouldn't this being true cause problems by it overriding the config the user provided if the user has set it to false?
I think this because the flags override the config.

Reply:
can address this in the pr that uses these flag values